integration/integration_test.py

```diff
@@ -908,19 +933,24 @@ def _temp_path():
         generate_decimal_case(),
         generate_datetime_case(),
         generate_nested_case(),
-        generate_dictionary_case()
+        generate_dictionary_case().expect_fail(
```

Comment:
Hmm, why? We should fix the failure instead...

Reply:
It's not supported yet: https://issues.apache.org/jira/browse/ARROW-3200

Reply:
Then perhaps simply skip it until it's supported? The "expected fail" machinery doesn't look very useful to me.

Reply:
Ok, sounds good.